### PR TITLE
Fix/#154 remove unused reports commands

### PR DESCRIPTION
**Pull request by shyam-nalluri**
#154

### CODE REVIEW

File: tests/__snapshots__/cli_test_commands/normal_base_help.txt [hunk 1]
```diff
@@ -5,8 +5,6 @@ Usage: test [OPTIONS]
 Options:
   --debug / --no-debug            Enables or disables debug mode
   -c, --config-file FILE          Pass external configuration file to Eze Cli
-  -s, --scan-type TEXT            named custom scan type to run aka production
```
Comment (mgonzales-rs) [inline tests/__snapshots__/cli_test_commands/normal_base_help.txt:8]: just wondering, do we need to delete this flag?

**Review by mgonzales-rs — APPROVED** [review]
left one comment, apart from that LGTM

File: tests/__snapshots__/cli_test_commands/normal_base_help.txt (hunk 1 above)
Comment (shyam-nalluri) [reply tests/__snapshots__/cli_test_commands/normal_base_help.txt:8]: I believe that we are no more using that flag

Comment (shyam-nalluri) [reply tests/__snapshots__/cli_test_commands/normal_base_help.txt]: Added back

**Comment by Anthony-Mckale** [comment]
fixed unit tests and removed the other commands

**Comment by Anthony-Mckale** [comment]
good job shyam

**Pull request merged**